committee-code-of-conduct/charter.md

```diff
@@ -147,4 +147,10 @@ self-elected reason, they must notify the committee as well as the steering
 committee in writing. As a courtesy, such notifications should be given at least
 30 calendar days in advance of their departure.
 
+### Participation in another committee
+If a current code of conduct committee member chooses to run for election in
+another Kubernetes communtiy committee, they are required to resign from the code
+of conduct committee upon the announcement of their candidacy (not after the election
+takes place).
```

Comment:
To align with the above paragraph about resignation, would it be better to say that they are required to announce their resignation from the CoCC with the resignation taking effect the date the election ends? That would allow for a notice period and at least a short overlap with their replacement for continuity purposes.

I think it would also be good to call out specifically that this applies only to committees that would cause a conflict of interest, since I don't think such a conflict of interest is present with the Security Response Committee.

Reply:
Yeah maybe at this time it makes sense to keep it specific to steering